tools/train.py

```diff
@@ -57,14 +57,12 @@ def train():
 
     trainer = Trainer(**config.trainer, logger=experiment_logger, callbacks=callbacks)
     logger.info("Training the model.")
-    trainer.fit(model=model, datamodule=datamodule)
 
-    logger.info("Loading the best model weights.")
-    load_model_callback = LoadModelCallback(weights_path=trainer.checkpoint_callback.best_model_path)
-    trainer.callbacks.insert(0, load_model_callback)
+    ckpt_path = config.trainer.get("ckpt_path", None)
+    trainer.fit(model=model, datamodule=datamodule, ckpt_path=ckpt_path)
```

Comment:
What's the reason for passing the checkpoint path during training?

Reply:
The idea is to allow the users to continue training from a checkpoint.

Reply:
Did you check if this works?

Reply:
I tried this with stfpm and it resumes training. But I noticed that this should be `config.get` rather than `config.trainer`. I'll push the fix. Thanks